omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/CustomDatatypeResource1_9.java

```diff
@@ -129,6 +130,9 @@ protected PageableResult doGetAll(RequestContext context) throws ResponseExcepti
 	}
 	
 	private List<CustomDatatypeRepresentation> getAllCustomDatatypes() {
+		if(!Context.isAuthenticated()) {
+			throw new APIAuthenticationException("User must be authenticated!");
+		}
```

Comment:
Same here, this looks unrelated, I think you are modifying another recent commit unintentionally?